app/components/LocationSubTitle/index.js

```diff
@@ -0,0 +1,115 @@
+/**
+*
+* SubTitle
+*
+*/
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import StyledSmall from 'components/shared/StyledSmall';
+import PageTitle from 'components/shared/PageTitle';
+import moment from 'moment';
+import {} from 'react-intl';
+
+import LocationHeader from 'components/LocationHeader';
+const LocationImage = styled.img`
+  width: 100%;
+  height: 128px;
+  object-fit: fill;
+`;
+
+const SubTitleOverlayDiv = styled.div `
+  background-color: rgba(54,64,88,.6); 
+`;
+
+const SubTitleHeader = styled.h2 `
+  font-family: AvenirNext-Regular;
+  font-style: normal;
+  font-size: 1.5em;
+  color: white;
+`;
+
+const SubTitleInfo = styled.ul `
+  font-family: AvenirNext-Regular;
+  font-style: normal;
+  font-size: 0.75em;
+  color: white;
+`;
+
+const CardBorderDiv = styled.div`
+    box-shadow: 2px 6px 8px 2px rgba(0, 0, 0, 0.2);
+`;
+
+
+function LocationSubTitleHelper(props) {
+  return (
+    <div>
+    <LocationImage
+            className="img-fluid"
+            src={props.image.url} height = "128"
+          />
+    </div>
+    );
+}
+
+LocationSubTitleHelper.propTypes = {
+  image: PropTypes.shape({
+    id: PropTypes.number.isRequired,
+    url: PropTypes.string.isRequired,
+  }).isRequired,
+};
+
+const LocationSubTitle = (props) => {
+  console.log(props.image)
+  return (
+    <CardBorderDiv>
+    <div className="position-relative">
```

Comment:
Replace this with `<CardBorderDiv className="position-relative">` and remove the `<CardBorderDiv>` above